services/gitdiff/gitdiff.go

```diff
@@ -577,9 +579,19 @@ parsingLoop:
 				break curFileLoop
 			case strings.HasPrefix(line, "old mode ") ||
 				strings.HasPrefix(line, "new mode "):
+
+				if strings.HasPrefix(line, "old mode ") {
+					curFile.OldMode = strings.ReplaceAll(line, "old mode ", "")
```

Comment:
I guess it should be `TrimPrefix` and then `TrimSpace`, because there is a line below `strings.HasSuffix(line, " 160000\n")`